Magento_GiftRegistry/templates/customer/list.phtml

```diff
@@ -0,0 +1,177 @@
+                                <a
+                                    href="#"
+                                    data-post='<?= /* @noEscape */ $block->getDeleteParams($item) ?>'
+                                    class="padding-right-xs"
+                                    title="<?= $block->escapeHtmlAttr(__('Delete')) ?>"
+                                    data-role="remove-gift-registry"
+                                >
+                                    <?= $block->escapeHtml(__('Delete')) ?>
+                                </a>
+                            </td>
+                        </tr>
+                    <?php endforeach; ?>
+                </tbody>
+            </table>
+        </div>
+    </div>
+    <?php if ($block->getChildHtml('pager')) :?>
+        <div
+            class="
+                toolbar
+                giftregistry-toolbar
+                bottom
+            "
+        >
+            <?= $block->getChildHtml('pager') ?>
+        </div>
+    <?php endif;?>
+<?php else :?>
+    <div class="message">
+        <?= $block->escapeHtml(__('You haven\'t created a gift registry yet.')) ?>
+    </div>
+<?php endif;?>
+
+<div class="actions-group">
+    <div class="actions-group__handler">
+        <?php if ($block->canAddNewEntity()) :?>
+            <a
+                href="<?= $block->escapeUrl($block->getAddUrl()) ?>"
+                class="
+                    link--button
+                    button
+                    actions-group__button
+                "
+                title="<?= $block->escapeHtmlAttr(__('Add New')) ?>"
+            >
+                <?= $block->escapeHtml(__('Add New')) ?>
+            </a>
+        <?php endif;?>
+    </div>
+    <div class="actions-group__handler">
+        <a
+            href="<?= $block->escapeUrl($block->getBackUrl()) ?>"
+            class="
+                link
+                actions-group__link
+            "
+            title="<?= $block->escapeHtmlAttr(__('Back to My Account')) ?>"
+        >
+            <?= $block->escapeHtml(__('Back')) ?>
+        </a>
+    </div>
+</div>
+<script>
+    require([
+        'mage/dataPost',
+        'Magento_Ui/js/modal/confirm',
+        'prototype'
+    ], function(dataPost, confirm){
+//<![CDATA[
+        $$("[data-role='remove-gift-registry']").each(function(item){
+            Event.observe(item, "click", function (event) {
+                var params = JSON.parse(jQuery(event.currentTarget).attr('data-post'));
+
+                event.stopPropagation();
+                confirm({
+                    content: '<?= $block->escapeJs($block->escapeHtml(__('Are you sure you want to delete this gift registry?'))) ?>',
+                    actions: {
+                        confirm: function () {
+                            dataPost().postData(params);
+                        }
+                    }
+                });
+            });
+        });
+//]]>
+    });
```

Comment:
I think we can fix this indentaiton and move whole things 4 spaces left (expect CDATA as it is already there)